packages/epo-widget-lib/src/widgets/CameraFilter/CameraFilter.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import { ComponentMeta, ComponentStoryObj } from "@storybook/react";
+import { Container } from "@rubin-epo/epo-react-lib";
+
+import CameraFilter from ".";
+
+const meta: ComponentMeta<typeof CameraFilter> = {
+  argTypes: {},
+  component: CameraFilter,
+  decorators: [
+    (Story) => (
+      <Container>
+        <Story />
+      </Container>
+    ),
+  ],
+};
+export default meta;
+
+export const Primary: ComponentStoryObj<typeof CameraFilter> = {
```

Comment:
there are no props for this widget, everything is statically bundled, so there is no additional Storybook config.